### PR TITLE
feat: Unescape the strings received from the syslog service

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
https://github.com/appium/appium/issues/14476

### CODE REVIEW

**Comment by mykola-mokhnach** [comment]
@KazuCocoa It would be cool if you cold help with testing on real devices

**Comment by KazuCocoa** [comment]
okay, will test this on a real device

**Comment by mykola-mokhnach** [comment]
@KazuCocoa did the fix work for your local tests?

File: lib/syslog/transformer/syslog-decoder.js [hunk 1]
```diff
@@ -2,6 +2,71 @@ import Stream from 'stream';
...
+  while (index < bytesView.length) {
+    if (bytesView[index] === BACKSLASH_CODE) {
+      const [newIndex, chars] = parseUtf8Chars(index);
+      if (newIndex > index) {
+        result += chars;
+        index = newIndex;
+        continue;
+      }
+    }
+    result += String.fromCharCode(bytesView[index++]);
```
Comment (KazuCocoa) [inline lib/syslog/transformer/syslog-decoder.js:64]: ah, i wondered fromCodePoint but https://stackoverflow.com/questions/34680898/can-you-use-string-fromcodepoint-just-like-string-fromcharcode

**Comment by KazuCocoa** [comment]
Yeah, I just tested this this morning

**Comment by mykola-mokhnach** [comment]
Thanks @KazuCocoa

**Pull request merged**